components/logins/android/src/main/java/mozilla/appservices/logins/DatabaseLoginsStorage.kt

```diff
@@ -29,6 +27,7 @@ import mozilla.components.service.glean.private.LabeledMetricType
  * LoginsStorage implementation backed by a database.
  */
 class DatabaseLoginsStorage(private val dbPath: String) : AutoCloseable, LoginsStorage {
+    private lateinit var store: PasswordStore
```

Comment:
This file can *almost* be removed - except for the locking. I made changes in b0d4c761 but `unlock()` seems wrong.